src/SFA.DAS.Roatp.CourseManagement.Web/Models/Standards/StandardDetailsViewModel.cs

```diff
@@ -12,11 +15,51 @@ public class StandardDetailsViewModel
         public string RegulatorName { get; set; }
         public string Version { get; set; }
         public string CourseDisplayName => $"{CourseName} (Level {Level})";
+
         public bool IsStandardRegulated => !string.IsNullOrEmpty(RegulatorName);
+
         public string StandardInfoUrl { get; set; }
         public string ContactUsPhoneNumber { get; set; }
         public string ContactUsEmail { get; set; }
         public string ContactUsPageUrl { get; set; }
+        public List<ProviderCourseLocationViewModel> ProviderCourseLocations { get; set; }
+
+        public List<ProviderCourseLocationViewModel> SubRegionCourseLocations { get; set; }
+       
+        public ProviderCourseLocationViewModel NationalCourseLocation { get; set; }
+
+        public List<string> Regions()
```

Comment:
Can be a get property

Reply:
I changes to a get, but SonarCloud identifies that as a critical codesmell, and said to convert to a method, so set back again.